src/pages/products/sgid/planning/water-related-landuse.astro

```diff
@@ -0,0 +1,87 @@
+import ExternalLink from '@components/page/ExternalLink.astro';
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah Water Related Land Use',
+  description: `Water usage-related data (irrigated crops, phreatophytes, wet/open water areas, dry land agriculture and residential/industrial areas).`,
+  stewards: ['DNR DWRe'],
+  type: ProductType.POLYGON,
+  category: SgidCategory.PLANNING,
+  secondaryCategory: SgidCategory.ENVIRONMENT,
+};
+
+const page: IPageMetadata = {
+  ...metadata,
+  updateHistory: [`The "SurveyYear" field contains currency information`],
+  hub: {
+    ...dataPages[metadata.title],
+  },
+};
+---
+
+<Layout title={page.title} subTitle={page.updateHistory[0]} category={page.category} actionUrl={page.application}>
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      Water usage-related polygon map data, published annually by the Utah Division of Water Resources at DNR that depicts the types and extent of irrigated crops, as well as information concerning phreatophytes, wet/open water areas, dry land agriculture and residential/industrial areas. The primary business driver for this dataset is for constructing and analyzing the state's annual water budget.
+      </p>
+  </Fragment>
+
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <Fragment slot="description">
+    <p>
+      As part of developing and updating the State Water Plan, the <ExternalLink href="https://water.utah.gov/">Utah Division of Water Resources</ExternalLink> collects water-related land use data for the entire state. This data includes the types and extent of irrigated crops
+      as well as information concerning phreatophytes, wet/open water areas, dry land agriculture and residential/industrial
+      areas.
+    </p>
+    <p>
+      The data produced by the water-related land use program are used for various planning purposes. Some of these
+      include: determining cropland water use, evaluating irrigated land losses and conversion to urban uses, planning
+      for new water development, estimating irrigated acreages for any area, and developing water budgets. Additionally,
+      the data are utilized by many other state and federal agencies. Water-related land use can be used to depict
+      Irrigation type (Irrigated, Sub-Irrigated, Non-Irrigated, Riparian, Urban, etc).
+    </p>
+    <p>
+      See the accompanying FGDC metadata for description of extensive Land Use, Irrigation Type, and Vegetation Cover
+      codes. There are no constraints or warranties with regard to the use of this dataset.
```

Comment:
**question**: does the FGDC metadata exist outside of sgid internal?

Reply:
Good catch, it may exist in AGOL but I'm just putting the relevant stuff in for kicks.